core/src/main/resources/hudson/model/ManageJenkinsAction/index.jelly

```diff
@@ -57,13 +57,14 @@ THE SOFTWARE.
               <div class="jenkins-section__item">
               <j:set var="alt" value="${icon.replaceAll('\\d*\\.[^.]+$', '')}"/>
               <j:set var="iconSrc" value="${h.tryGetIconPath(m.iconFileName, context)}"/>
+              <j:set var="sure" value="${%sure}"/>
               <j:set var="iconXml">
                 <l:icon src="${m.iconFileName}" />
               </j:set>
-              ${taskTags!=null and attrs.contextMenu!='false' ? it.addContextMenuItem(taskTags, m.urlName, iconSrc, iconXml, m.displayName, m.requiresPOST, m.requiresConfirmation, m.badge) : null}
+              ${taskTags!=null and attrs.contextMenu!='false' ? it.addContextMenuItem(taskTags, m.urlName, iconSrc, iconXml, m.displayName, m.requiresPOST, m.requiresConfirmation, m.badge, sure) : null}
               <j:choose>
                 <j:when test="${m.requiresConfirmation}">
-                  <l:confirmationLink href="${m.urlName}" post="${m.requiresPOST}" message="${%are.you.sure(m.displayName)}">
+                  <l:confirmationLink href="${m.urlName}" post="${m.requiresPOST}" message="${%sure}" title="${m.displayName}">
```

Comment:
When I click this it jumps me to the top of the page which is a bit jarring. There's a flash as it happens.

Reply:
I think the reason is that confirmationlink renders an `<a href="#" ...`
This makes the browser jump to the top. I think I've seen this behaviour even before.

Reply:
https://stackoverflow.com/questions/2084750/javascript-anchor-avoid-scroll-to-top-on-click